mono/w32subset.h

```diff
@@ -0,0 +1,616 @@
+/**
```

Comment:
Shouldn't this be in a subdirectory ?

Reply:
Put it there since it is included by pretty much all libraries so didn't know if we would have a good place for things that could be shared that way, but maybe we could put it in eglib?

Reply:
Either eglib/ or utils/ would be good. eglib/ is no longer separate, so it can include stuff from utils/ i think.

Reply:
I have seen includes from utils in eglib, https://github.com/mono/mono/blob/cd4ccd7e56fe637d995653c2f4278ace5b7723ba/mono/eglib/gdir-unix.c#L33, so guess it is fine to put it in utils and include it from there.